lib/matplotlib/ticker.py

```diff
@@ -2204,10 +2219,19 @@ def _decade_greater(x, base):
     return greater
 
 
+@_api.deprecated("3.6")
 def is_close_to_int(x, *, atol=1e-10):
     return abs(x - np.round(x)) < atol
 
 
+def _is_close_to_int(x, *, atol=None, rtol=None):
```

Comment:
None of the code above uses the tolerances, so we can keep it simple and leave them out
```suggestion
def _is_close_to_int(x):
    return math.isclose(x, round(x))
```

Note also that the current implementation ignores *rtol* if *atol* is given. This is different from e.g. numpy and could lead to surprises if we start using that in the future. But as said, since we don't need it, let's leave it out. We can always add it later if needed and then worry about the correct implementation.

Reply:
Sure.

Reply:
(The previous code passed rtol as atol, which was really my original concern.)